From 5e57be3762a9dc72eb3b1d22b45d9a3e449e9d35 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 25 Oct 2023 14:45:32 +0200
Subject: [PATCH 0681/1141] ARM: dts: stm32: add spi6 SCMI clock / reset for
 stm32mp15 boards

Update the clock / reset properties for spi6 node in case of
scmi based boards.

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: I72956c7f681567ef6ec48db77dff951838521d53
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/336139
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm/boot/dts/stm32mp157a-ed1-scmi.dtsi | 5 +++++
 arch/arm/boot/dts/stm32mp157c-ed1-scmi.dtsi | 5 +++++
 2 files changed, 10 insertions(+)

--- a/arch/arm/boot/dts/stm32mp157a-ed1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157a-ed1-scmi.dtsi
@@ -84,6 +84,11 @@
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
 
+&spi6 {
+	clocks = <&scmi_clk CK_SCMI_SPI6>;
+	resets = <&scmi_reset RST_SCMI_SPI6>;
+};
+
 &usart1 {
 	clocks = <&scmi_clk CK_SCMI_USART1>;
 };
--- a/arch/arm/boot/dts/stm32mp157c-ed1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157c-ed1-scmi.dtsi
@@ -89,6 +89,11 @@
 	clocks = <&scmi_clk CK_SCMI_RTCAPB>, <&scmi_clk CK_SCMI_RTC>;
 };
 
+&spi6 {
+	clocks = <&scmi_clk CK_SCMI_SPI6>;
+	resets = <&scmi_reset RST_SCMI_SPI6>;
+};
+
 &usart1 {
 	clocks = <&scmi_clk CK_SCMI_USART1>;
 };
